File: Programming Basics/JavaScript/While loops/Labs/min-number.js
function solve(input) {
    let min = Number.POSITIVE_INFINITY;

    let index = 0;
    while (input[index] != "Stop") {
        const currentNumber = Number(input[index]);
        if (currentNumber < min) min = currentNumber;

        index++;
    }

    console.log(min);
}

File: Programming Basics/C++/While loops/Labs/MinNumber/MinNumber.cpp
#include <iostream>
#include <limits.h>
#include <string>
using namespace std;

int main()
{
    int min = INT_MAX;

    string input;
    while (cin >> input && input != "Stop")
    {
        int currentNumber = stoi(input);
        if (currentNumber < min) min = currentNumber;
    }

    cout << min << endl;
}
